recipes/cartools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "cartools" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 090875bed4491af3bf0a5e501ebd7fbc201272b3db4f736506fe63b9b202b848
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
```

Comment:
Please change this to `noarch: python` so conda knows to move things around during deployment.